dev/com.ibm.ws.transaction/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -78,3 +78,12 @@ manual=Manual
 
 default.max.shutdown.delay=Default maximum shutdown delay
 default.max.shutdown.delay.desc=Default maximum shutdown delay
+
+lease.check.strategy=Lease check strategy
+lease.check.strategy.desc=The algorithm used by the recovery service to check the recovery lease log
+
+lease.check.interval=Lease check interval
+lease.check.interval.desc=The interval after which the lease check strategy is executed
+
+lease.length=Lease length
+lease.length.desc=The length of time after which a recovery log lease expires
```

Comment:
**change**
The length of time after which a recovery log lease expires

**to**
The length of time before a recovery log lease expires